docs/users/students.md

```diff
@@ -22,36 +24,41 @@ To add new students to a course, click the _Add New Students_ button at the top-
 
 The _title_ and _affiliation_ fields are used across the platform to better identify students.
 
+**Team Name**: If you're adding a student to a team, you can enter the team's name here. If you're not adding the student to a team, leave this field blank.
+
+Students with same team name will be grouped together; this will not affect existing teams in the course.
+
 **Tags**: Tags are keywords or strings to associate with the list of students that you're adding, and can be used to quickly filter the list of students in a course. You can start typing and pick from the suggested list of tags or create new ones.
 
 **Notify students**: Use this checkbox to enable or disable onboarding email notification for the newly added students.
 
 ### Add multiple students at once
 
 1. Fill in the details of the first student, and click the _Add to List_ button.
-2. This will add the first student to a list that is unsaved, and preserve the _Title_, _Affiliation_, and _Tags_, so that you can add more students to the list.
+2. This will add the first student to a list that is unsaved, and preserve the _cohort_, _Title_, _Affiliation_, and _Tags_, so that you can add more students to the list.
```

Comment:
Match capitalization.

```suggestion
2. This will add the first student to a list that is unsaved, and preserve the _Cohort_, _Title_, _Affiliation_, and _Tags_, so that you can add more students to the list.
```